src/components/Badge/Badge.js

```diff
@@ -2,26 +2,99 @@ import PropTypes from 'prop-types';
 import styled, { css } from 'react-emotion';
 import { size } from 'polished';
 
+import { values } from '../../util/fp';
```

Comment:
Sorry, I thought I read elsewhere that you and fernando agreed to use lodash for better browser support? And here you're using the custom FP functions? I'm confuzzled.